.gitignore

```diff
@@ -19,6 +19,7 @@ cabal.sandbox.config
 cabal.project.local
 
 .vscode
+.idea/
```

Comment:
No worries on this, just curious: is there a good IDEA extension for Haskell?

Reply:
Haha good eye! I didn't look yet, but yes this came up b/c it was the first time I'd opened the project in a JetBrains-developed editor. I ran into a merge conflict when I rebased before merging makefile-tool, and I'm just used to the interactive merge interface from PyCharm so went w/ that.